responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css

```diff
@@ -0,0 +1,224 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #dfdfe2;
+}
+
+.cat-container {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto 0;
+}
```

Comment:
Forgot to mention this earlier, but there doesn't seem to be an element with this class. These can be removed if they're not going to be used:

```suggestion
```